content/ar/docs/contribute/localization.md

```diff
@@ -24,4 +24,30 @@ content_type: concept
 
 English | العربية
 --- | --- 
-Kubernetes |  كوبيرنيتيس
+Kubernetes (k8s) | (ك8س) كوبيرنيتيس
+Abstraction | تجريد
+Agile Software Development | تطوير مرن للبرمجيات
+Auto Scaling | توسع آلي
+Cloud native technologies | تقنية الحوسبة السحابية الأصيلة
+Cluster | عنقود
+Container Image | صورة الحاوية
+Container | حاوية
+Containerization | تحوية
+Node | عقدة
+Pod | حجيرة
+Bare-metal machine | جهاز حقيقي مجرد
+Scaling | تحجيم
+Virtaul Machine | جهاز افتراضي
+Machine | جهاز
+Dependency | تبعيات
+Load Balancer | موزع الحمل
+Minikube | مينيكيوب
```

Comment:
```suggestion
Minikube | مينيكيوب
`minikube` | `minikube`
```

Sometimes the command needs to stay unchanged.

Reply:
This glossary is a reference to unify the Arabic translation of the technical works I do not think that we should include commands or code

Reply:
Whatever you find useful!

Reply:
I agree with @sftim on the idea here. Let's at least mention explicitly that code and commands should stay untranslated (I know it may be obvious to a developer, but perhaps not a beginner!)